### DISCUSSION TITLE
how to use queue2csv.sh?

### DISCUSSION THREAD

**Discussion by ricardo-kcc** [AFLplusplus/AFLplusplus#1572] [General]
the error as below.
![image](https://user-images.githubusercontent.com/64991294/198821966-cff78b5e-82cb-41a9-9360-c4a9430d54ac.png)

and afl++ command is ` ./afl-fuzz -m none -d -i ../jpeg -o ./out -- ./jhead_afl @@`

**Comment by vanhauser-thc** [comment]
`utils/analysis_scripts/queue2csv.sh out/default foo.csv "./jhead_afl @@"`

**Reply by ricardo-kcc** [reply]
the error still exists after execute the command.

**Reply by vanhauser-thc** [reply]
maybe a permission problem? it works fine so it is something on your system/setup
